docroot/modules/custom/my_assets/my_assets.module

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for my_assets module.
+ *
+ * @DCG
+ * This file is no longer required in Drupal 8.
+ * @see https://www.drupal.org/node/2217931
+ */
+
+function my_assets_element_info_alter(array &$types)
+{
+  if (isset($types['table'])) {
+    $types['table']['#attached']['library'][] = 'my_assets/custom-tables';
+  }
```

Comment:
this will add to all tables but what if we want this only for specific tables?

Reply:
It is in the requirement to add to all tables.
Specific table can be customized from theme.